mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/ProtoUtils.java

```diff
@@ -53,11 +68,26 @@ private ProtoUtils() {}
             .build();
     }
 
+    /**
+     * Utility method to make logging of protos a little better than what is done by default.
+     * @param message    The proto message to be "stringified"
+     * @return  A string representation of the provided {@code message} surrounded with curly braces ('{}') and all
+     *          new lines replaced with a space.
+     */
     @NotNull
-    public static String protoToString(@NotNull final Message message) {
+    public static String protoToString(@NotNull final Object message) {
```

Comment:
There was a scenario (`State` iirc where I couldn't easily ensure that all the types being passed to this method were `Message`s. So I broadened the scope back to object, since it's really only useful for debug logging (which I hope we can get rid of, I thought it was okay.